docker_scylla/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM scylladb/scylla
```

Comment:
How is this dockerfile supposed to be used?

Reply:
The running of the test in the CI currently uses the docker system. This is done for the ScyllaDB and DynamoDB.

When running a basic docker image obtained such as `scylladb/scylla` then we are running with default settings. In particular, this exposes us to the tombstone problem.

It is important to see how we can bypass the problem by increasing the size of the tombstone allowed. The variable in question is `query_tombstone_page_limit`. To do that, we need to change the input file `scylla.yaml`. The file is located in `/etc/scylla/scylla.yaml` in the docker image.

The way to do this change is to create a new image. And so this require a `Dockerfile`. The building of the image from the Dockerfile is instantaneous since only a file is changed. The ENTRYPOINT is not changed by this operation.

Reply:
In summary the purpose of this Dockerfile is to demonstrate how we can change the tombstone limitation by changing the input to the `scylla` database.

Reply:
Gotcha, this is used in CI.

Reply:
It would be nice to avoid creating a top-level directory for this. Could it go to `.github`?